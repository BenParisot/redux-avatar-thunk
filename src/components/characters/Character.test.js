import React from 'react';
import Character from './Character';
import { shallow } from 'enzyme';

describe('Character component', () => {
  it('renders a character component', () => {
    const wrapper = shallow(<Character photoUrl="./image.png" name="Avatar-Man" _id="12345" />); 
    expect(wrapper).toMatchSnapshot();
  });
});
