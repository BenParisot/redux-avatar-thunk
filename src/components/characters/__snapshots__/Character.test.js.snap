// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders a character component 1`] = `
<Fragment>
  <p>
    Avatar-Man
  </p>
  <img
    src="./image.png"
  />
</Fragment>
`;
